lab-scott/test/fs-modules-tests.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const expect = require('chai').expect;
+const Buffer = require('buffer').Buffer;
+const fs = require('fs');
+const fsMod = require('../lib/fs-module');
+let expectedArr = [ '5468652053656168', '48656c6c6f2e205a', '476f204d27732061' ];
+fsMod.reads(fsMod.hexArr);
+
+describe('fs module', function(){
+  describe('#reads', function(){
+    it('should read file one, and produce a raw buffer', done => {
+      fs.readFile(`${__dirname}/../data/one.txt`, function(err, data){
+        let bufferData = data;
+        expect(typeof bufferData).equal(typeof Buffer.from(''));
+        done();
+      });
+    });
+    it('should read file two, and produce a raw buffer', done => {
+      fs.readFile(`${__dirname}/../data/two.txt`, function(err, data){
+        let bufferData = data;
+        expect(typeof bufferData).equal(typeof Buffer.from(''));
+        done();
+      });
+    });
+    it('should read file three, and produce a raw buffer', done => {
+      fs.readFile(`${__dirname}/../data/three.txt`, function(err, data){
+        let bufferData = data;
+        expect(typeof bufferData).equal(typeof Buffer.from(''));
+        done();
+      });
+    });
+    it('should read three files in order and confirm they are in order', done => {
+
+      for (let i = 0; i<3; i++){
+
```

Comment:
I see what you are trying to do here. It might be easier to have a read file to run the for loop in then use the data as an array to pass it through. I have yet to see a for loop in a test. You could also you expect the indexes in order in three lines, expect[0] to be value at that index.